tools/lab_dispatch/resources/service_file.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
```

Comment:
There might be a reason for this filename that I don't know about, but would it be possible to rename the file to something like `checkbox-agent.service` or `checkbox-ng.service` or anything that is suggestive of which service it describes?

Reply:
no reason, turning it into checkbox-ng.service

Reply:
Actually, removing this, as this is no longer needed. It was needed when I wanted to provision fully from source, but now that I use debs, I get the service "for free"